### DISCUSSION TITLE
find deleted file information

### DISCUSSION THREAD

**Discussion by xiaoweihong** [seaweedfs/seaweedfs#3975] [Q&A]
how can i find deleted information from volume file?
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/13499653/201653686-1cd3f1de-136b-4054-b1d7-237fc6be680a.png">
sth like fid etc

**Comment by chrislusf** [comment]
what is your use case?

**Comment by xiaoweihong** [comment]
some file can not find,want to analyse and find
